Source/NSTableColumn.m

```diff
@@ -609,7 +615,9 @@ - (id) initWithCoder: (NSCoder*)aDecoder
 
 	  if (version >= 4)
 	    {
+	      _tableView = [aDecoder decodeObject]; // not retained, tableView retains us...
```

Comment:
Did you add another value to the encoded format without increasing the version number? Hopefully nobody did create an archive with the intermediate format.

Reply:
I considered this, but I think it's likely okay because we didn't do a release in the interim.   The new decoding is because the methods for registering the prototype views are on NSTableView according to Apple documentation.